iroha-lib/model/generators/CommandGenerator.cpp

```diff
@@ -232,14 +232,14 @@ std::shared_ptr<Command> CommandGenerator::generateSetAccountQuorum(
 std::shared_ptr<Command> CommandGenerator::generateSubtractAssetQuantity(
         const std::string& asset_id,
         const std::string& amount,
-        const std::string& title)
+        const std::string& description)
 {
     SubtractAssetQuantity subtractAssetQuantity;
     subtractAssetQuantity.set_asset_id(asset_id);
     subtractAssetQuantity.set_amount(amount);
-    std::optional<std::string> title_optional = title;
-    if (title_optional) {
-        subtractAssetQuantity.set_title(*title_optional);
+    std::optional<std::string> description_optional = description;
+    if (description_optional) {
+        subtractAssetQuantity.set_description(*description_optional);
     }
```

Comment:
if will be always true, change as it was for AddAssetQuantity:
```suggestion
    if (! description.empty()) {
        subtractAssetQuantity.set_description(description);
    }
```